conf/messages.en

```diff
@@ -442,4 +446,25 @@ date.month.8 = August
 date.month.9 = September
 date.month.10 = October
 date.month.11 = November
-date.month.12 = December
\ No newline at end of file
+date.month.12 = December
+
+charityRegistrationNumber.title = What is your charity registration number (CHRN)?
```

Comment:
This one should be: 

What is your charity registration number (CHRN)? - register for the Economic Crime Levy - GOV.UK

Reply:
Resolving this based on Pete's comment bellow.